ebay-frontend/src/app/upcoming-events-screen/upcoming-events-screen.component.scss

```diff
@@ -1,7 +1,57 @@
 .upcoming-card {
+        background-color: #FFFFFF;
+        display: flex;
+        justify-content: space-between;
+       
+
+}
+.left {
+  background: white;
+}
+.right {
+  background: white;
+  color: gray;
+  line-height: 100px;
```

Comment:
`line-height: 100px` doesn't scale very well. Try `align-items: center` to make the text stay in the middle